package.json

```diff
@@ -66,6 +66,9 @@
     ],
     "*.robot": [
       "python3 -m pipenv run robotidy --config tests/robot-tests/robotidy.toml"
+    ],
+    "*.py": [
+      "python3 -m pipenv run autopep8 --max-line-length 120 -r --in-place tests"
```

Comment:
I think this line shouldn't have `tests` at the end - as this runs it against the entire tests directory, rather than the specific python files that have changed.